docs/source/index.rst

```diff
@@ -0,0 +1,21 @@
+.. records-mover documentation master file, created by
+   sphinx-quickstart on Fri Jun 26 11:46:58 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to records-mover's documentation!
+=========================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   records_mover
+
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```

Comment:
I was sort of expecting documentation generation to be entirely driven by docstring comments, but the Sphinx toolkit seems to be a tad more rustic than that.  As a result, these .rst files contain:

* any tweaks that need to be done to the default generation strategy (you'll see some examples later on)
* lists of the specific public API that will be documented
* any module-level documentation needed, which doesn't seem to be brought in at all (I generally chose to leave this out and preferred documenting things at the class level)